app/controllers/charges_controller.rb

```diff
@@ -24,6 +27,39 @@ def create
 
   private
 
+  def create_order_for_vendors
```

Comment:
Assignment Branch Condition size for create_order_for_vendors is too high. [20.25/15]